### PR TITLE
stretch has been moved to archive.debian.org

### PR DESCRIPTION
**Pull request by fabikd**
<hr/>

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Control Cache Config (`t3c`, formerly ORT)
- Traffic Control Health Client (tc-health-client)
- Traffic Control Client 
- Traffic Monitor
- Traffic Ops
- Traffic Portal
- Traffic Router
- Traffic Stats
- Grove
- CDN in a Box
- Automation 
- unknown

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

The stretch has been moved to archive.debian.org and in broke the cdn-in-a-box build of traffic_stats, this fixes it! (:

### CODE REVIEW

File: infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 FROM influxdb:1.7.3
 
+RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
+
```
Comment (zrhoffman) [inline infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb:21]: Might as well just update to [`influxdb:2.7.0`](https://hub.docker.com/layers/library/influxdb/2.7.0/images/sha256-32b2c4d5e6fc96da0f6eb325aacc635964d44844cac95bbdee07e620afdf27e4?context=explore), which uses Bullseye

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb (hunk 1 above)
Comment (fabikd) [reply infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb:21]: I've tried to update to version 2.7.0, and it breaks some of the js for the ui in tp.

Comment (zrhoffman) [reply infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb:21]: Ah okay. Does `influxdb:1.8.10` work? If not, we should at least be able to upgrade to `influxdb:1.7.11`.

Comment (zrhoffman) [reply infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb:21]: >> I've tried to update to version 2.7.0, and it breaks some of the js for the ui in tp.
>
> Ah okay. Does `influxdb:1.8.10` work? If not, we should at least be able to upgrade to `influxdb:1.7.11`.

`influxdb:1.8.10` uses Bullseye, but `influxdb:1.7.11` still uses Stretch. So if `influxdb:1.8.10` does not work, we'll still need your `sources.list` fix.

**Review by zrhoffman — APPROVED** [review]
Approving, let's look at updating the image in a later PR. Thanks for your contribution!

**Pull request merged**